### ISSUE TITLE
New cursor move up and down (alt + ctrl + up || down)

### ISSUE THREAD

**Issue by benschac** [kankaristo/atom-multi-cursor-plus#4]
## Need

```gherkin
As a user
I want up and down of new cursors to work.
So that I can actually use multicursor.
```

HALP.

**Comment by kankaristo** [comment]
I think this is a duplicate of #2?

Is the new version (v1.0.1) working for you?

**Comment by benschac** [comment]
Just download it yesterday for the first time.  I think so.

On Tuesday, January 19, 2016, Sami Kankaristo <notifications@github.com>
wrote:

> I think this is a duplicate of #2
> <https://github.com/kankaristo/atom-multi-cursor-plus/issues/2>?
>
> Is the new version (v1.0.1) working for you?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kankaristo/atom-multi-cursor-plus/issues/4#issuecomment-172865450>
> .
>


-- 
Sent from Gmail Mobile

**Comment by kankaristo** [comment]
I'll close this, since this should be fixed in v1.0.1.

Please comment here if the problem continues, and I'll reopen.
